test/validation/api/thread/thread.c

```diff
@@ -81,20 +82,30 @@ static int thread_global_term(odp_instance_t inst)
 
 static void thread_test_odp_cpu_id(void)
 {
-	(void)odp_cpu_id();
-	CU_PASS();
+	CU_ASSERT(odp_cpu_id() >= 0);
 }
 
 static void thread_test_odp_thread_id(void)
 {
-	(void)odp_thread_id();
-	CU_PASS();
+	int id = odp_thread_id();
+
+	/* First created thread */
+	CU_ASSERT(id == 0);
```

Comment:
This is not guaranteed by the API spec.

Reply:
We discussed about this last week with Petri and the intention of the API has always been this. However, you are correct that the API spec is not  completely clear on this. This check (and another similar in `thread_test_odp_thread_count()`) could be removed for now and added back when the spec is clarified.

Reply:
Fixed in V2.